fastlane/Fastfile

```diff
@@ -143,11 +144,10 @@ platform :ios do
   #####################################################################################
   desc 'Updates the AppStoreStrings.pot file with the latest data'
   lane :update_appstore_strings do |options|
-    project_root = File.dirname(File.expand_path(__dir__))
```

Comment:
This was an unnecessary redeclaration of `PROJECT_ROOT_FOLDER` (see [L14](https://github.com/woocommerce/woocommerce-ios/pull/6678/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R14))